src/components/List/List.jsx

```diff
@@ -49,6 +45,8 @@ export const List = ({ token }) => {
     }
   }, [value]);
 
+  console.log(items);
```

Comment:
Another nit - I'm sorry because I often leave console logs in too, haha! Seeing another on line 60, as well.